apps/ensnode/src/api/index.ts

```diff
@@ -1,16 +1,59 @@
-import { db } from "ponder:api";
+import { db, publicClients } from "ponder:api";
 import schema from "ponder:schema";
+import { ponderMetadata } from "@ensnode/ponder-metadata/middleware";
 import { graphql as subgraphGraphQL } from "@ensnode/ponder-subgraph/middleware";
 import { Hono } from "hono";
+import { cors } from "hono/cors";
 import { client, graphql as ponderGraphQL } from "ponder";
+import packageJson from "../../package.json";
+import {
+  ensNodePublicUrl,
+  getEnsDeploymentChain,
+  ponderDatabaseSchema,
+  requestedPluginNames,
+} from "../lib/ponder-helpers";
 
 const app = new Hono();
 
+// use CORS middleware
+app.use(
+  cors({
+    origin: "*",
+  }),
+);
+
+// use root to redirect to the ENSAdmin website with the current server URL as ensnode parameter
+app.use("/", async (ctx) =>
+  ctx.redirect(`https://admin.ensnode.io/about?ensnode=${ensNodePublicUrl()}`),
+);
```

Comment:
Takes the public URL of ENSNode and goes to ENSAdmin UI to show that ENSNode stats to the user.